src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const populationList = document.querySelectorAll('.population');
+const totalInner = document.querySelector('.total-population');
+const averageInner = document.querySelector('.average-population');
+
+function getTotal() {
+  let sum = 0;
+
+  for (let i = 0; i < populationList.length; i++) {
```

Comment:
It's better to replace the `for` loop with the reduce method to calculate the sum of population values. This provides a more concise and readable solution.